ethereum/contracts/IncentivizedOutboundChannel.sol

```diff
@@ -7,10 +7,21 @@ import "./OutboundChannel.sol";
 // IncentivizedOutboundChannel is a channel that sends ordered messages with an increasing nonce. It will have incentivization too.
 contract IncentivizedOutboundChannel is OutboundChannel {
 
+    uint256 gasFee;
+    address feeController;
+
     constructor() {
         nonce = 0;
+        gasFee = 0;
```

Comment:
I think it might make sense here to pass `gasFee` as an argument to the constructor.